rllib/tuned_examples/simple_q/cartpole_simpleq_test.py

```diff
@@ -0,0 +1,10 @@
+from ray.rllib.algorithms.simple_q import SimpleQConfig
+
+
+config = (
+    SimpleQConfig()
+    .framework("tf")
+    .rollouts(num_rollout_workers=0)
+    .environment(env="CartPole-v1")
```

Comment:
Up to you, but I feel like doing `.environment("CartPole-v1")` looks nicer (without the `env=`).